versioning.md

```diff
@@ -0,0 +1,29 @@
+Versioning our libraries
+========================
+
+Which repositories?
+-------------------
+
+Some of our repositories are used as libraries and included as a requirement in other repositories.
+eg https://github.com/OpenDataServices/flatten-tool/releases
```

Comment:
Is the /releases page the correct one to link to here?

Reply:
Well, it's just an example link, and given the whole point of this page is released versions then releases seems like a good page to link to.